test/Dialect/FIRRTL/annotations.fir

```diff
@@ -98,11 +100,12 @@ circuit Foo: %[[{"one":null,"target":"~Foo|Foo>bar.a"},
   module Foo:
     inst bar of Bar
 
-    ; CHECK-LABEL: module {
+    ; CHECK-LABEL: firrtl.circuit "Foo"
+    ; CHECK: firrtl.module @Bar
+    ; CHECK-SAME: [#firrtl<"subAnno<fieldID = 2, {circt.nonlocal = @\22~Foo|Foo/bar:Bar>b.qux\22, three}>">]
```

Comment:
What's going on with the `\22` ASCII code instead of the `"` in the test after this? 🤔